SwanOauthRenew/README.md

```diff
@@ -1,6 +1,8 @@
 # SwanOauthRenew
 
-Extension that refreshes the Oauth token 
+This is a server extension that fetches the oAuth tokens available in the "user" endpoint of the JupyterHub API (1) and stores them in files that can be access by other processes (like the EOS client). It can be configured to take any token from the auth state dictionary and write it with any content format.
```

Comment:
`that can be accessed`

Reply:
Can you elaborate a bit on the purpose of this extension after the modifications? It does not refresh the oauth token anymore? What's the use of storing them in files? Are the other processes that read the files use the tokens for authentication themselves?

Reply:
Before this extension was looking for a single token (which was in a known location inside a dict), and storing it in a file (configured via env variables). We only had one token back then, which was wrong, and now we have more tokens, 1 per service we need to connect to.

What the new extension does is removing all the hardcoded stuff. So now you can say where to look in the dict, and where to store the final file. Basically, in the first version it was hardcoded for EOS, now it's configurable for any service we need. 

But the _raison d'être_ of the extension is still the same: pick a token that JH renewed, and store it in a file where running processes can look.